updates/35/updates.sql

```diff
@@ -91,4 +91,40 @@ INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x643C4E15d7d6
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:8453/erc20:0xd173C2C9386AEE88b1Ca06522C951be67cbf38CC', 'KONTOS', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:8453/erc20:0xd173C2C9386AEE88b1Ca06522C951be67cbf38CC', 'A', 8453, '0xd173C2C9386AEE88b1Ca06522C951be67cbf38CC', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:8453/erc20:0xd173C2C9386AEE88b1Ca06522C951be67cbf38CC', 'KOS', 'kontos', '', NULL, 1733718089, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('ZEREBRO', 'Zerebro', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('ZEREBRO', 'ZEREBRO', 'zerebro', 'ZEREBRO', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:8453/erc20:0xB1a03EdA10342529bBF8EB700a06C60441fEf25d', 'Mister Miggles', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:8453/erc20:0xB1a03EdA10342529bBF8EB700a06C60441fEf25d', 'A', 8453, '0xB1a03EdA10342529bBF8EB700a06C60441fEf25d', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:8453/erc20:0xB1a03EdA10342529bBF8EB700a06C60441fEf25d', 'MIGGLES', 'mister-miggles', 'MIGGLES', NULL, 1721147639, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('MUMU', 'MUMU THE BULL', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('MUMU', 'MUMU', 'mumu-the-bull-3', 'MUMU', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:56/erc20:0x76A797A59Ba2C17726896976B7B3747BfD1d220f', 'Wrapped TON Coin', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:56/erc20:0x76A797A59Ba2C17726896976B7B3747BfD1d220f', 'A', 56, '0x76A797A59Ba2C17726896976B7B3747BfD1d220f', 9, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:56/erc20:0x76A797A59Ba2C17726896976B7B3747BfD1d220f', 'TONCOIN', 'the-open-network', 'TON', NULL, 1633015603, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('FWOG', 'Fwog', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('FWOG', 'FWOG', 'fwog', 'FWOG', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('MCG', 'MetalCore', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('MCG', 'MCG', 'metalcore', 'MCG', NULL, NULL, NULL);
+*
```

Comment:
those two are mising the timestamp but you can check them by checking when the contracts where deployed in solscan